modules/site/src/app/examples/pagination/pagination-split-usage.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../ng2-material/core/style/variables";
+@import "../../../ng2-material/core/style/default-theme";
+@import "../../../ng2-material/components/whiteframe/whiteframe";
+
+pagination-split-usage {
```

Comment:
Done. I've chosen  to display only one page on small screen. A good way to show that you can change pagination depending on media queries too.